Cargo.toml

```diff
@@ -50,17 +50,20 @@ unicode-xid = "0.2"
 unscanny = "0.1"
 usvg = { version = "0.22", default-features = false }
 xmp-writer = "0.1"
+tracing = "0.1.37"
 indexmap = "1.9.3"
+stacker = "0.1.15"
 
 [profile.dev]
-debug = 0
+debug = true
 
 [profile.dev.package."*"]
 opt-level = 2
 
 [profile.release]
 lto = "thin"
 codegen-units = 1
+debug = true
```

Comment:
Maybe create a separate build profile, something like `profile.perf`?

Reply:
That's not even required for tracing, that's an artifact of doing VTuner profiling yesterday to help @dccsillag 👍

Reply:
Oh, do you mean that this will be removed before merge?